board/piksiv3/rootfs-overlay/usr/bin/dump_core.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+/usr/bin/logger "CORE DUMP!! executable=$1 pid=$2 time=$3"
+echo "CORE DUMP!! executable=$1 pid=$2 time=$3" | /usr/bin/sbp_log --error
+
+corefile="/tmp/cores/core-$1-$2-$3"
+touch $corefile
+
+while read line
+do
```

Comment:
The core dumps have a mask where you say what sections to dump. Now with the mask 0x23 the core dumps are very tiny (less than 100 Kbytes).

Also, I set the core dump size to "unlimited", we can actually say 1 Mbyte or so.